DataHub/UserInterface/02b_AttributeSets.md

```diff
@@ -0,0 +1,397 @@
+
+- ![Settings](../../Assets/Icons/Settings01.png "[Settings]") (Settings)   
+  Click this button to unlock the *Required* column in the attributes list. The color of the button switches to blue if the column is unlocked. Click the button again to lock the *Required* column. The button color switches back to gray.
+
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+  Click this button to display the search bar and search for an attribute.
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+  Click this button to update the list of attributes.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+  Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all attributes in the list are selected.
+
+- ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit)   
+  Click this button to edit the configuration of the selected attribute. This button is only displayed if a single checkbox in the list of attributes is selected and if the selected attribute has an editable configuration. The *Edit attribute for set "Attribute set name"* view is displayed. Alternatively, you can click directly a row in the list to edit an attribute configuration.
+
+[comment]: <> (Edit is not working -> Bugticket)
+
+- ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete)   
+  Click this button to remove the selected attribute from the attribute set. This button is only displayed if at least one checkbox of an attribute is selected.       
+
+  > [Info] If you remove an attribute from the attribute set, the attribute itself is not deleted, but it is unassigned to the selected attribute set.   
+
+- [ADD MAPPING]   
+  Click this button to add a mapping to the selected attribute. The *Add mapping* view is displayed. You can only add a mapping for attribute sets that are not inherited from another attribute set. This button is only displayed if a single checkbox of an attribute is selected.
+
+
+The list displays all attributes assigned to the selected attribute set. Depending on the settings, the displayed columns may vary.   
+If the ![Settings](../../Assets/Icons/Settings01.png "[Settings]") (Settings) button is gray, all fields are read-only. If the ![Settings](../../Assets/Icons/Settings02.png "[Settings]") (Settings) button is blue, the field *Required* is editable.
+
+- *Name*   
+  Attribute name.
+
+
+- *Calculated field*   
+  Indication whether the attribute is calculated by an ETL mapping from other fields to itself or not:
```

Comment:
```suggestion
  Indication whether the attribute is calculated by an ETL mapping from other fields to itself or not:
```
itself hört sich für mich hier komisch an... "from other fields to this one/the current/selected one"?